docs/src/main/sphinx/release/release-383.md

```diff
@@ -0,0 +1,113 @@
+# Release 383 (1 June 2022)
+
+## General
+
+* Improve query performance by reducing worker-to-worker communication overhead. ({issue}`11289`)
+* Improve performance and reduce memory usage of queries that contain aggregation. ({issue}`12336`)
+* Add IAM Role support for exchange spooling on S3. ({issue}`12444`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+## Accumulo connector
+
+* Deny `CREATE TABLE` and `CREATE TABLE AS SELECT` statements if a table comment is specified. ({issue}`12452`)
```

Comment:
why would that be .. because the connector does not support table comments ? Is there a useful error message?

Reply:
> because the connector does not support table comments ?

Yes

>  Is there a useful error message?

It throws "This connector does not support creating tables with table comment" after this change.

Reply:
Also ... its not so much of a denial.. more a fail ..

Fail table creation when table comments are specified, because table comments are not supported by the connector.

maybe...

Reply:
Add something like "Previously, the comment was ignored."